modules/client/src/allowlistVoting/client.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  AllowlistVoting as AllowlistVotingContract,
+  AllowlistVoting__factory,
+} from "@aragon/core-contracts-ethers";
+import { ClientCore } from "../client-common";
+import { AllowlistVotingContextPlugin } from "./context";
+import { AllowlistVotingDecoding } from "./internal/decoding";
+import { AllowlistVotingEncoding } from "./internal/encoding";
+import { AllowlistVotingEstimation } from "./internal/estimation";
+import { AllowlistVotingMethods } from "./internal/methods";
+
+export class AllowlistVoting extends ClientCore {
```

Comment:
Why renaming the class? We have agreement in naming this plugin AddressList, even if contract+subgraph name changes have not happened yet.

Reply:
renamed back to `addresslist`